discretisedfield/field.py

```diff
@@ -444,3 +447,48 @@ def _writeovf(self, filename, representation="txt"):
         f.write(footerstr)
 
         f.close()
+
+
+    def _get_colormap(self, name, n):
+        data = []
+        cmap = pylab.cm.get_cmap(name, n)
+        for i in range(cmap.N):
+            rgb = cmap(i)[:3]
+            data.append(rgb)
+
+        return np.array(data)
+
+
+    def plot3d(self, vector_scale=1.0, direction=0, color=0xff, colormap_name='', **kwargs):
+
+        plot = k3d.plot()
+
+        # value of vectors
+        vectors = self.array.reshape(-1,3)
```

Comment:
PEP8 wants a space after a comma. Please run 'pep8' or 'flake' on this to make the source PEP8 compatible.

Reply:
I agree. Running:
```
pycodestyle ovf2vtk.py
```
checks the PEP8. (`pep8` still exists, but `pycodestyle` is a new name for the package: https://github.com/PyCQA/pycodestyle/issues/466)